src/UI/Components/Diagram/components/ErrorsContainer.tsx

```diff
@@ -13,7 +13,7 @@ import { RelationCounterForCell } from "../interfaces";
  *
  * @returns {React.FC | null} The rendered validation component or null if no missing relations are found or the canvas is not edited/dirty.
  */
-export const Validation: React.FC = () => {
+export const ErrorsContainer: React.FC = () => {
```

Comment:
Maybe ErrorMessageCollapsible/ErrorMessageContainer would be a bit more explicit